exchangelib/folders.py

```diff
@@ -117,6 +117,12 @@ def __init__(self, id, changekey):
         self.id = id
         self.changekey = changekey
 
+    @property
+    def item_id(self):
```

Comment:
Unless I'm missing something, `ItemId` isn't supported as an item by services as it's used in most cases:

```python
>>> from exchangelib.folders import ItemId
>>> item = ItemId("someid", "somechangekey")
>>> item_id = ItemId(*(item if isinstance(item, tuple) else (item.item_id, item.changekey)))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'ItemId' object has no attribute 'item_id'
```

Hence the reason adding this. However, I agree. This isn't the best way of doing it. If I was to try again, I'd possibly try a more general `ItemId` constructor, but that's definitely out of the scope of this PR, so I'll delete this to revert to original behaviour.